### DISCUSSION TITLE
Hooks always come out 2.5mm instead of 2.2mm...

### DISCUSSION THREAD

**Discussion by tym0t0r** [aderusha/DDD-Printable-Wall-Control-System#68] [Q&A]
When printing hooks and side pieces, the elements that go through the WallControl slots always come out too thick/tall for me: 2.5mm vs 2.2mm. I know this is likely a general slicer or printer issue, but I've asked in other forums and haven't been able to find a solution. I'm wondering if anyone here has run into the something similar, and if so, what they did about it.

**Printer**
Artillery Sidewinder X2
I've flashed an updated firmware to Marlin 2.1
Limited hardware mods (and I don't imagine they'd be a cause, but listing for completeness):

- [Z-Axis Stabiliser Brackets](https://www.thingiverse.com/thing:5320520)
- Bi-metal heat break
- 0.6mm nozzle

**Material**
eSun PLA+

**Slicer**
SuperSlicer (happy to share particular settings, but I'm not sure what is relevant).

Calibration Steps Taken

I found and like Michael's [Teaching Tech](https://teachingtechyt.github.io/calibration.html) calibration pages and videos and have done my best to follow and complete most of them. These seem like the most relevant ones he touches on:

- Extruder E-steps Calibration - done
- Slicer Flow Calibration - done
- XYZ steps Calibration - done

In all my searching I also found Vector3D's [Calilantern](https://vector3d.co.uk/product/calilantern/) tool for calibration, bought it, and have gone through that process. Based on instructions, I've done one print and set of measurements and made the prescribed adjustments within my slicer and firmware. (I need to do another follow-on print and measurement still for confirmation. Just printed last night, but haven't had time to remeasure.)

Still not getting the results, I happened upon SuperSlicer's own calibration menu and steps. Some of them were repeats of Michael's but I went through them anyways. All good until I get to the last check, the dimensional cubes. This is where I am seeing the consistent +0.3mm variation (when measuring 20 and 30mm cubes that I print). Just to be clear the X and Y axis/sides are measuring at the expected values (or within a smaller margin of error that I'm ok with).

Other Searching
Most references I've been able to find only speak to prints coming out too short. From the [Slic3r Manual](https://manual.slic3r.org/troubleshooting/dimension-errors) (which I understand to be the underpinnings of SuperSlicer):

> If your vertical dimensions are wrong (i.e. along the Z axis) -and your object is usually shorter than expected- it means your nozzle is too low, thus the first layer is pressed too much on the print bed. To fix this, you might want to raise your Z endstop or increase the Z offset option in Slic3r.

If the inverse is true, then my nozzle is too high...? This can't be. I've leveled my bed and zeroed my Z settings with a feeler gauge. I can't go any lower without going into my bed--certainly not .3mm lower.

___

I'm sure the end solution is going to be something simple, but I feel like I don't know where else to look. Thanks for any help!

**Comment by Jameson42** [comment]
What layer height (and first layer height) are you printing at?

**Comment by tym0t0r** [comment]
I tried 0.3mm layer height previously. I've switched to 0.2mm, but that hasn't solved it. First layer height also set to 0.2mm currently. I also tried it set as a percentage (75%), but that didn't seem to change the outcome either.

**Comment by mrcpu99** [comment]
Thoughts:

  Might try printing outerwalls first, which at least with cura, helps with
dimensional accuracy in some cases.

  It's interesting that .6 nozzle with 4 lines would land at 2.4mm, and you
want 2.3, but getting 2.5, which almost sounds like either your printer
isn't printing thin lines, or the slicer is erring on the side of "too
much" instead of too little.  So if it's not printing thin lines, but
printing 4 full width lines, then perhaps it makes sense.

  I'd try a single-wall version, and measure that.

**Reply by tym0t0r** [reply]
Just tried single-wall and then another as single-wall + outerwalls first. Perhaps a slight improvement on XY, but no change on Z. 😢

**Comment by mrcpu99** [comment]
What layer height are you printing at?

**Reply by tym0t0r** [reply]
Most recently, 0.2mm. I've tried previously at 0.3mm and got similar results.

**Comment by Hercules117** [comment]
I know this is unhelpful, but I always have to shrink down the hook portion by -.2mm to get mine to fit too.

**Comment by jensbodal** [comment]
0.24mm seemed to be the sweet spot for me with a 0.6 nozzle. Though if I can't print with the flats/hooks/pegs laying down horizontally and instead have to print them vertically, I tend to get larger tabs.  Though after trying out Orca Slicer and doing my best to copy settings over I'm seeing too large again.

**Comment by aderusha** [comment]
I'll note that printing vertically is almost certain to result in poor strength.  The tab design was specifically designed in the orientation provided to allow for maximum strength.

To the OP: one other solution is just to grab a small file and open up the slot a touch, takes only a couple seconds/few strokes.
